tkg/tkgconfigbom/bom.go

```diff
@@ -471,12 +471,17 @@ func (c *client) DownloadTKGCompatibilityFileFromRegistry(repo, resource string,
 
 	// begin download of compatibility file
 	tkgCompatibilityImagePath := fmt.Sprintf("%s/%s", repo, resource)
-	log.Infof("Downloading TKG compatibility file from '%s'", tkgCompatibilityImagePath)
+	log.Infof("Listing TKG compatibility tags from '%s'", tkgCompatibilityImagePath)
```

Comment:
I wonder if printing "listing compatibility tags" would be confusing to a user since that list will not be printed to the screen (I think).  The listing part seems to be an implementation detail that the user would not be aware of.

But just my inexperienced 2-cents

Reply:
Stick it under a V(6) or something?

Reply:
I was thinking that the original message of "Downloading TKG compatibility" made sense to the user as they understand that there is a download going on.  The listing was an "obscure" step however.

How about reverting the original messages and adding a V(6) (or whatever level you feel appropriate) for the implementation-specific steps?

Reply:
+1 to put this behind a log level. L505 probably suffice in happy path.
Rest of changes lgtm.